stable/velero/templates/deployment.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.configuration.provider -}}
 {{- $provider := .Values.configuration.provider -}}
-apiVersion: apps/v1beta2
```

Comment:
This is inconsistent with Velero's upstream versions.

@skriss what do you think about getting these RBAC and deployment versions onto v1 for 1.0?

Reply:
yeah makes sense, users can always tweak manually if they're running on old versions.